pkg/controller/acmechallenges/update.go

```diff
@@ -35,7 +34,7 @@ import (
 	"github.com/cert-manager/cert-manager/test/unit/gen"
 )
 
-var argumentError = stderrors.New("invalid arguments")
+var argumentError = errors.New("invalid arguments")
```

Comment:
nitpick: If we use `fmt.Errorf` rather than `errors.New` here we can save an import and produce a trivial speedup of our builds 😂 

Obviously not a blocker but I wouldn't personally keep the `errors` import just for one use case here.